### ISSUE TITLE
Some parser features are not discoverable or reflected in usage/doc

### ISSUE THREAD

**Issue by cmnbroad** [broadinstitute/barclay#35]
-list files/comments in list files
-collection argument files/comments (https://github.com/broadinstitute/barclay/pull/28)
-using "null" to reset collection arguments
-argument tagging rules (https://github.com/broadinstitute/barclay/pull/33)

Its probably too much to include in usage output every time. We may want to add a special command line option to display this kind of help, or just output a link to an online resource describing these things.

**Comment by cmnbroad** [comment]
@vdauwera Let us know you thoughts on this.

**Comment by vdauwera** [comment]
I agree these topics should be covered elsewhere. That's already the case in GATK3.
